quiz_app/quiz_app/tests/test_questions_views.py

```diff
@@ -0,0 +1,35 @@
+import pytest
```

Comment:
We should test not only a happy path, but errors also.
For example: 404 error you catched if the question does not exist, or there is no data in POST request.